### PR TITLE
Correct name Christiaan

### PR DESCRIPTION
**Pull request by cniweb**
Signed-off-by: Christian Häussler <c-n-i@web.de> (github: @cniweb)

### CODE REVIEW

File: docs/input/credits.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
----
+a---
```
Comment (gep13) [inline docs/input/credits.md]: Can you remove this?

**Comment by gep13** [comment]
@cniweb well spotted!

**Comment by gep13** [comment]
@cniweb are you in a position to sign the cla for this repo?

File: docs/input/credits.md (hunk 1 above)
Comment (cniweb) [inline docs/input/credits.md]: Yes

**Comment by gep13** [comment]
@cniweb thanks for making this change!

**Pull request merged**